app/Providers/BroadcastServiceProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\Facades\Broadcast;
+use Illuminate\Support\ServiceProvider;
+
+class BroadcastServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap any application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        Broadcast::routes();
+
+        require base_path('routes/channels.php');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File is being conditionally included; use "include" instead](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)